scm/driver/bitbucket/webhook.go

```diff
@@ -70,6 +70,8 @@ func (s *webhookService) Parse(req *http.Request, fn scm.SecretFunc) (scm.Webhoo
 		if hook != nil {
 			hook.(*scm.IssueCommentHook).Action = scm.ActionDelete
 		}
+	case "repo:commit_status_updated", "repo:commit_status_created":
```

Comment:
should we ignore statuses that are not pipeline executions?  I think we can use `commit_status.type=="build"`

Reply:
added the check for build type